hera_cal/io.py

```diff
@@ -1812,28 +1812,20 @@ def baselines_from_filelist_position(filename, filelist, polarizations=None):
         name of the file being processed.
     filelist : list of strings
         name of all files over which computations are being parallelized.
-    polarizations : list of strings
```

Comment:
why remove the polarizations functionality here? don't have a strong opinion just wondering what the motive is

Reply:
Splitting different polarizations in the same baseline into different files runs into trouble downstream at the stage where power spectra waterfalls for different baselines are combined into single files again. `hera_pspec` does not currently support combining power-spectrum files with the same baseline times but different polarizations so it made sense to avoid splitting polarizations.